config/README.md

```diff
@@ -0,0 +1,29 @@
+# Purpose of module
+Enable configuration support and the possibility to `@Inject` configuration classes where needed.
+
+# How
+Configuration provided in a yml file is read, and the properties are mapped to configuration classes annotated with the `@Config`
+annotation. Configuration file is pointed out by the last argument passed when starting the `Main.class`
+There is also support for putting environment variables into the configs by setting the config value to an environment variable name 
+embraced with double curly braces.
+It will fetch the value from the environment by using the name you provide in the configuration file. 
+Useful when running the application in a container environment
```

Comment:
Missing trailing dot. No newlines needed above as the sentences are about the same thing?